### PR TITLE
refactor: remove fontawesome

### PR DESCRIPTION
**Pull request by goldenmaya**
Description:
https://virtocommerce.atlassian.net/browse/ST-4361

--

QA-test:

Demo-test:

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.38.0-pr-700-9542-95421660.zip

### CODE REVIEW

File: client-app/pages/forgot-password.vue [hunk 1]
```diff
@@ -5,13 +5,15 @@
         v-t="'pages.forgot_password.header'"
         class="mb-8 text-3xl font-bold uppercase tracking-wide lg:mt-5 lg:text-4xl"
       ></h1>
-      <div v-if="!isSucceeded">
+
+      <div v-if="!isSucceeded && false">
```
Comment (NaMax66) [inline client-app/pages/forgot-password.vue:9]: false?

File: client-app/shared/catalog/components/category-selector.vue [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
+    <div class="mb-1.5 h-5 bg-[--color-neutral-50]">&nbsp;</div>
+    <div v-for="i in 6" :key="i" class="mb-1.5 ml-6 h-4 bg-[--color-neutral-50]">&nbsp;</div>
   </VcFilterCardSkeleton>
 
   <VcFilterCard v-else :full-width-content="!category?.childCategories?.length">
     <template v-if="!!parentCategory" #header>
       <router-link
         :to="getCategoryRoute(parentCategory!)"
-        class="-mx-1.5 grow rounded-sm px-1.5 py-0.5 text-[color:var(--color-filter-card-header)] hover:bg-gray-100"
+        class="-mx-1.5 flex grow items-center gap-1.5 rounded-sm px-1.5 py-0.5 hover:bg-[--color-neutral-50]"
```
Comment (NaMax66) [inline client-app/shared/catalog/components/category-selector.vue:11]: Should we really use -mx?

Comment (goldenmaya) [reply client-app/shared/catalog/components/category-selector.vue:11]: возможно, не знаю, не стала пока трогать тут верстку.

File: client-app/shared/catalog/components/vendor.vue [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
-      <i class="fas fa-star mr-0.5 text-primary" />
-      <span class="font-extrabold">{{ vendor.rating?.value }}</span
-      >/5&nbsp;({{ vendor.rating?.reviewCount }})
+  <div>
+    <div class="font-bold text-[--color-accent-600]">{{ vendor.name }}</div>
+
+    <div v-if="$cfg.rating_enabled && displayRating" class="flex items-center">
+      <VcIcon class="me-0.5 text-[--color-primary-500]" name="star" size="xs" />
+
+      <span class="text-xs font-black"> {{ vendor.rating?.value }} </span>/5 ({{ vendor.rating?.reviewCount }})
```
Comment (asvishnyakov) [inline client-app/shared/catalog/components/vendor.vue:8]: Don't remove non-breaking space or add no-wrap

**Review by NaMax66 — APPROVED** [review]

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**